marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.tsx

```diff
@@ -158,7 +158,7 @@ const MappingCard: React.FC<Props> = (props) => {
         width={350}
         maskClosable={false}
         >
-        <span style={{fontSize: '16px'}}>Are you sure you want to delete this?</span>
+        <span style={{fontSize: '16px'}}>Are you sure you want to delete the <b>{mappingArtifactName}</b> step?</span>
```

Comment:
@AkshayBajajML Run tile uses `<strong>` and the same gets verified in the e2e test with "contains.text" assertion.. Try using `<strong>` instead of `<b>` here with the assertion I shared, to see if it works for e2e tests.